app.py

```diff
@@ -79,9 +79,28 @@ def vote(vote_button, response_a, response_b, model_a_name, model_b_name,
 }
 """
 
-with gr.Blocks(title="Arena", css=css) as app:
+with gr.Blocks(title="Yanolja Arena", css=css) as app:
   set_token(app)
 
+  with gr.Row():
+    gr.HTML("""
+    <h1 style="text-align: center; font-size: 28px; margin-bottom: 16px">Yanolja Arena</h1>
+    <p style="text-align: center; font-size: 16px">Yanolja Arena helps find the best LLMs for summarizing and translating text. We compare two random models at a time and use an ELO rating system to score them.</p>
+    <p style="text-align: center; font-size: 16px">This is an open-source project. Check it out on <a href="https://github.com/yanolja/arena">GitHub</a>.</p>
+    """)
+  with gr.Accordion("How to Use", open=False):
+    gr.Markdown("""
+      1. **For Summaries:**
+        - Type or paste the text you want summarized into the prompt box.
```

Comment:
I think `paste` is redundant, it's just one way of entering texts. same for `type`.
what about `Enter the text you ...`?